pkg/openstack/loadbalancer.go

```diff
@@ -606,6 +607,22 @@ func (lbaas *LbaasV2) getLoadBalancerLegacyName(_ context.Context, _ string, ser
 	return cloudprovider.DefaultLoadBalancerName(service)
 }
 
+// Returns a list of custom load balancer listener tags for the specified  service.
+func (lbaas *LbaasV2) getCustomLoadBalancerTags(service *corev1.Service) []string {
+	var listenerTags []string
+	annotationVal := getStringFromServiceAnnotation(service, ServiceAnnotationLoadBalancerCustomTags, "")
+
+	listOfTags := strings.Split(annotationVal, ",")
+
+	for _, tags := range listOfTags {
+		tags = strings.TrimSpace(tags)
```

Comment:
I think you can save some lines and a var by doing it like this:

```
    tags := strings.Split(annotationVal, ",")
	for i, tag := range tags {
		listOfTags[i] = strings.TrimSpace(tag)
	}
    return tags
```

Reply:
https://go.dev/play/p/fiUy3r7Arg9

i came up with this following your suggestion